tests/unit/TickingMockClockTest.php

```diff
@@ -28,7 +28,9 @@ public function it_returns_time_that_has_ticked(): void
         // Immutable variant.
 
         $sample1 = $clock->now();
+        \usleep(1);
```

Comment:
😭

Reply:
Seems like `nowMutable` costs an extra cycle. Can't make it to fail in the test, so not adding crude `usleep`s there